src/lib/pay-request/services/admin_users/types.ts

```diff
@@ -3,13 +3,7 @@ export enum GoLiveStage {
   EnteredOrganisationName = 'ENTERED_ORGANISATION_NAME',
   EnteredOrganisationAddress = 'ENTERED_ORGANISATION_ADDRESS',
   ChosenPSPStripe = 'CHOSEN_PSP_STRIPE',
-  ChosenPSPWorldpay = 'CHOSEN_PSP_WORLDPAY',
```

Comment:
Just wondering why the Worldpay entries are being removed?
And I can't see a gov banking option?

Reply:
For government banking, stages are `CHOSEN_PSP_GOV_BANKING_WORLDPAY` and `TERMS_AGREED_GOV_BANKING_WORLDPAY`. Looks like this list is not up to date